spec/controllers/openid_connect/authorization_controller_spec.rb

```diff
@@ -32,50 +32,6 @@
         stub_sign_in user
       end
 
-      context 'service providers requires AAL3' do
```

Comment:
did these specs get moved somewhere else?

Reply:
Ah yes, I did not like taking these out. The `stub_sign_in_method` unfortunately stubs `confirm_two_factor_authenticated` too much to test these well here. Cleaning that up seemed like too much for this PR so I moved the tests into the `aal3_required_spec`s